cyber/common/global_data.cc

```diff
@@ -67,6 +67,8 @@ GlobalData::GlobalData() {
   is_reality_mode_ =
       config_.run_mode_conf().run_mode() == proto::RunMode::MODE_REALITY;
 
+  is_octopus_mode_ = false;
```

Comment:
Please refer to https://github.com/ApolloAuto/apollo/pull/11131#discussion_r415933639
We'd better avoid such change in Cyber, unless approved by @unacao and @vlin17.

Reply:
better not use code name/project name as variable name here. 
it's hard to understand what "octopus" is.  poor readability